server/routes/visitJourney/checkYourBooking.test.ts

```diff
@@ -226,13 +229,39 @@ testJourneys.forEach(journey => {
               username: 'user1',
               applicationReference: visitSessionData.applicationReference,
               applicationMethod: visitSessionData.requestMethod,
+              allowOverBooking: false,
             })
 
             expect(visitSessionData.visitStatus).not.toBe('BOOKED')
             expect(visitSessionData.visitReference).toBe(journey.isUpdate ? 'ab-cd-ef-gh' : undefined)
             expect(auditService.bookedVisit).not.toHaveBeenCalled()
           })
       })
+
+      describe('Handle API errors', () => {
```

Comment:
Could include the test above (`'should handle booking failure, display error message and NOT record audit event'`) to this `Handle API errors` block as it's basically just testing for any non-422 API error.